ci/docs-builder.py

```diff
@@ -109,6 +109,14 @@ def main():
     all_content_index_rst = os.sep.join([WORKING_DIR, 'pulp', 'docs', 'all_content_index.rst'])
     copyfile('docs/all_content_index.rst', all_content_index_rst)
 
+    # make the _templates dir
+    layout_dir = os.sep.join([WORKING_DIR, 'pulp', 'docs', '_templates'])
```

Comment:
This was a good question. I hadn't thought much about it, but after reading some I don't want the behavior that a path component will be replaced by an absolute path. I don't want that behavior because in several places paths are meant for a remote machine which has a different absolute path. `os.sep` doesn't do this.